<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-21T15:16:17 IST" name="Suite" finished-at="2024-08-21T15:16:51 IST" duration-ms="33891">
    <groups>
    </groups>
    <test started-at="2024-08-21T15:16:17 IST" name="Test_Chrome" finished-at="2024-08-21T15:16:27 IST" duration-ms="9951">
      <class name="Testcase.Verified_Home_Testcase2">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:Testcase.Verified_Home_Testcase2@4fb0f2b9]" started-at="2024-08-21T15:16:17 IST" name="setup" finished-at="2024-08-21T15:16:21 IST" duration-ms="3831" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Verified_Home_Page()[pri:0, instance:Testcase.Verified_Home_Testcase2@4fb0f2b9]" started-at="2024-08-21T15:16:21 IST" name="Verified_Home_Page" finished-at="2024-08-21T15:16:27 IST" duration-ms="6072" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Title does not match the expected Title.! expected [Crowd Funding] but found [Login | Minimal UI]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Title does not match the expected Title.! expected [Crowd Funding] but found [Login | Minimal UI]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at ObjectFile.Home_page2.Home(Home_page2.java:46)
at Testcase.Verified_Home_Testcase2.Verified_Home_Page(Verified_Home_Testcase2.java:11)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verified_Home_Page -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Testcase.Verified_Home_Testcase2@4fb0f2b9]" started-at="2024-08-21T15:16:27 IST" name="tearDown" finished-at="2024-08-21T15:16:27 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Testcase.Verified_Home_Testcase2 -->
    </test> <!-- Test_Chrome -->
    <test started-at="2024-08-21T15:16:27 IST" name="Test_Firefox" finished-at="2024-08-21T15:16:40 IST" duration-ms="12808">
      <class name="Testcase.Verified_Home_Testcase2">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:Testcase.Verified_Home_Testcase2@6f1de4c7]" started-at="2024-08-21T15:16:27 IST" name="setup" finished-at="2024-08-21T15:16:34 IST" duration-ms="6762" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Verified_Home_Page()[pri:0, instance:Testcase.Verified_Home_Testcase2@6f1de4c7]" started-at="2024-08-21T15:16:34 IST" name="Verified_Home_Page" finished-at="2024-08-21T15:16:40 IST" duration-ms="6034" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Title does not match the expected Title.! expected [Crowd Funding] but found [Login | Minimal UI]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Title does not match the expected Title.! expected [Crowd Funding] but found [Login | Minimal UI]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at ObjectFile.Home_page2.Home(Home_page2.java:46)
at Testcase.Verified_Home_Testcase2.Verified_Home_Page(Verified_Home_Testcase2.java:11)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verified_Home_Page -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Testcase.Verified_Home_Testcase2@6f1de4c7]" started-at="2024-08-21T15:16:40 IST" name="tearDown" finished-at="2024-08-21T15:16:40 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Testcase.Verified_Home_Testcase2 -->
    </test> <!-- Test_Firefox -->
    <test started-at="2024-08-21T15:16:40 IST" name="Test_Edge" finished-at="2024-08-21T15:16:51 IST" duration-ms="11078">
      <class name="Testcase.Verified_Home_Testcase2">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:Testcase.Verified_Home_Testcase2@4d5d943d]" started-at="2024-08-21T15:16:40 IST" name="setup" finished-at="2024-08-21T15:16:45 IST" duration-ms="5032" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Verified_Home_Page()[pri:0, instance:Testcase.Verified_Home_Testcase2@4d5d943d]" started-at="2024-08-21T15:16:45 IST" name="Verified_Home_Page" finished-at="2024-08-21T15:16:51 IST" duration-ms="6039" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Title does not match the expected Title.! expected [Crowd Funding] but found [Login | Minimal UI]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Title does not match the expected Title.! expected [Crowd Funding] but found [Login | Minimal UI]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at ObjectFile.Home_page2.Home(Home_page2.java:46)
at Testcase.Verified_Home_Testcase2.Verified_Home_Page(Verified_Home_Testcase2.java:11)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verified_Home_Page -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Testcase.Verified_Home_Testcase2@4d5d943d]" started-at="2024-08-21T15:16:51 IST" name="tearDown" finished-at="2024-08-21T15:16:51 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Testcase.Verified_Home_Testcase2 -->
    </test> <!-- Test_Edge -->
  </suite> <!-- Suite -->
</testng-results>
